ar_track_alvar/nodes/.vscode/c_cpp_properties.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I would guess this is a hidden VS code file that got hidden in the mix. Is that correct?